app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -222,7 +222,7 @@ class PatientSummaryScreen(
 
   private fun doneClicks() = doneButton.clicks().map { PatientSummaryDoneClicked(screenKey.patientUuid) }
```

Comment:
Since we are no longer changing the `ProgressMaterialButton` to `MaterialButton` let's change the commit message to `Rename doctorButton to teleconsultButton`.